cogs/challenges.py

```diff
@@ -194,10 +204,23 @@ async def on_message(self, message):  # Submitted role.
                 embed.set_footer(
                     text=f"#ID: {message.author.id} • {len(code)} chars • Language: {filetype}"
                 )
+                self.code_length[message.author.id] = (filetype, len(code))
+                
                 await submission_channel.send(embed=embed)
 
         elif message.channel.id in [
             680851798340272141,
             713841395965624490,
         ]:  # Automatic reaction
             await message.add_reaction("🖐️")
+        elif message == 't.char':
+            em = discord.Embed(description="Char Count", color=0x36393E)
+            self.code_length = dict(sorted(self.code_length.items(), key=lambda x: x[1][1]))
+                
+            if self.code_length == {}:
+                return await message.channel.send("No submissions yet")
+
+            for key, value in self.code_length.items():
+                em.add_field(f"`{value[1]}` <@{key}>", value[0], inline=True)
```

Comment:
This probably isn't the best formatting, fields take a lot of space. You could just put it in the embed description, for example:
```suggestion
            em.description = "\n".join([f"`{length}` | <@{user_id}> ({filetype})" for user_id, (filetype, length) in self.code_length.items()])
```